manage_breast_screening/notifications/services/application_insights_logging.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+
+from azure.monitor.opentelemetry import configure_azure_monitor
+
+
+class ApplicationInsightsLogging:
+    def __init__(self) -> None:
+        self.logger_name = os.getenv(
+            "APPLICATIONINSIGHTS_LOGGER_NAME", "manbrs-notifications"
+        )
+        self.logger = self.getLogger()
+
+    def configure_azure_monitor(self):
+        if os.getenv("APPLICATIONINSIGHTS_IS_ENABLED", "False") == "True":
+            # Configure OpenTelemetry to use Azure Monitor with the
+            # APPLICATIONINSIGHTS_CONNECTION_STRING environment variable.
+            configure_azure_monitor(
+                # Set the namespace for the logger in which you would like to collect telemetry for if you are collecting logging telemetry. This is imperative so you do not collect logging telemetry from the SDK itself.
+                logger_name=self.logger_name,
+            )
+        else:
+            default_logger = logging.getLogger(__name__)
+            default_logger.info("Application Insights logging not enabled")
+
+    def getLogger(self):
+        return logging.getLogger(self.logger_name)
+
+    def raise_an_exception(self, exception_name: str):
```

Comment:
Maybe
```suggestion
    def exception(self, exception_name: str):
```
as we aren't raising anything in this method.